a10_octavia/tests/unit/controller/worker/tasks/test_nat_pool_tasks.py

```diff
@@ -0,0 +1,160 @@
+#    Copyright 2020, A10 Networks
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+
+import copy
+import imp
+
+try:
+    from unittest import mock
+except ImportError:
+    import mock
+
+from octavia.common import data_models as o_data_models
+
+from a10_octavia.common.data_models import VThunder
+import a10_octavia.controller.worker.tasks.nat_pool_tasks as task
+from a10_octavia.tests.common import a10constants
+from a10_octavia.tests.unit import base
+
+VTHUNDER = VThunder()
+LB = o_data_models.LoadBalancer(id=a10constants.MOCK_LOAD_BALANCER_ID)
+
+
+class TestHandlerNatPoolTasks(base.BaseTaskTestCase):
+
+    def setUp(self):
+        super(TestHandlerNatPoolTasks, self).setUp()
+        imp.reload(task)
+        self.client_mock = mock.Mock()
+
+    def tearDown(self):
+        super(TestHandlerNatPoolTasks, self).tearDown()
+
+    def test_create_nat_pool_with_nat_pool_not_exists_with_nat_pool_flavor(self):
+        flavor = {u'nat_pool': {u'start_address': u'172.16.1.101', u'end_address': u'172.16.1.108',
```

Comment:
Since this is common, can become global variable as `FLAVOR` and then can make a deep copy of it and use here.

Reply:
ok